config/protector.php

```diff
@@ -33,7 +33,7 @@
     | 8: minute
     |
     */
-    'fileName' => '%1$s %4$4d-%5$02d-%6$02d %7$02d-%8$02d.sql',
+    'fileName' => '%1$s %4$4d-%5$02d-%6$02d %7$02d-%8$02d-%9$s.sql',
```

Comment:
gb.test 2022-03-03 13-50-6220b991cbdc9

i was just wondering how i got to 62 seconds. if we already use spaces here, we may as well use a space instead of a dash

Reply:
done